log_anonymizer.py

```diff
@@ -279,48 +357,81 @@ def randomize_numbers(number, is_ip_address = False):
+                return c
+            c += chunk.count("\n")
+
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(description="Anonymize log files. Anonymized files and their lookup tables are storedd in the `anonymized-logs` folder")
-    parser.add_argument('input_directory', type=str, help='Directory containing the log files to be anonymized')
-    parser.add_argument('output_directory', type=str, help='Directory where the anonymized files will be stored')
-    parser.add_argument('--ip', action='store_true', help='Only Anonymize IP Addresses')
-    parser.add_argument('--endpoint', action='store_true', help='Only Anonymize Endpoints')
-    parser.add_argument('--user', action='store_true', help='Only Anonymize User IDs')
-    parser.add_argument('--timestamps', action='store_true', help='Only Anonymize Timestamps')
+    parser = argparse.ArgumentParser(
+        description="Anonymize log files. Anonymized files and their lookup tables are storedd in the `anonymized-logs` folder"
+    )
+    parser.add_argument(
+        "input_directory",
+        type=str,
+        help="Directory containing the log files to be anonymized",
+    )
+    parser.add_argument(
+        "output_directory",
+        type=str,
+        help="Directory where the anonymized files will be stored",
+    )
+    parser.add_argument("--ip", action="store_true", help="Only Anonymize IP Addresses")
+    parser.add_argument(
+        "--endpoint", action="store_true", help="Only Anonymize Endpoints"
+    )
+    parser.add_argument("--user", action="store_true", help="Only Anonymize User IDs")
+    parser.add_argument(
+        "--timestamps", action="store_true", help="Only Anonymize Timestamps"
+    )
 
     args = parser.parse_args()
 
-    base_directory = args.input_directory 
+    base_directory = args.input_directory
     output_directory = args.output_directory
     os.makedirs(output_directory, exist_ok=True)
 
-    for filename in os.listdir(base_directory):
-        if not filename.endswith(tuple(files_to_exclude)):
-            input_file_path = os.path.join(base_directory, filename)
-            output_file_path = os.path.join(output_directory, f'anonymized_{filename}')
-            lookup_file_path = os.path.join(output_directory, f'lookup_table_{filename}.txt')
+    for file_name in tqdm.tqdm(os.listdir(base_directory), unit=' Files'):
+        if not file_name.lower().endswith(tuple(FILES_TO_EXCLUDE)):
+            input_file_path = os.path.join(base_directory, file_name)
+            output_file_path = os.path.join(output_directory, f"anonymized_{file_name}")
+            lookup_file_path = os.path.join(
+                output_directory, f"lookup_table_{file_name}.txt"
+            )
             lookup_table = {}
-
-            with open(input_file_path, 'r') as input_file, open(output_file_path, 'w') as output_file:
-                for line in input_file:
+            line_count = count_lines(filename=file_name)
```

Comment:
```suggestion
            line_count = count_lines(file_path=input_file_path)
```
I am running into issues on this line, python cannot find the file name. 
I think we need to provide the file path instead. This should make it work.

Reply:
This is a rebase deal. Yes, your patch suggestions are valid! :)